app/routes/index/tickets.js

```diff
@@ -4,7 +4,7 @@ import AuthenticatedRouteMixin from 'simple-auth/mixins/authenticated-route-mixi
 export default Ember.Route.extend(AuthenticatedRouteMixin, {
   model() {
     let userId = this.session.get('secure.user._id');
-    return this.store.query('ticket', { userId: userId });
+    return this.store.findAll('ticket', { userId: userId });
```

Comment:
This anyway should be done using `query` method, because we need to fetch results for concrete user. Find all doesn't use second parameter.